launcher/actions/initial_spikes_generator.xml

```diff
@@ -3,12 +3,32 @@
     <co_simulation_action>
         <title>Initial Spikes Generators</title>
         <description>Creating an NumPy array format file filled by zeroes</description>
+        <variables>
+            <var_000>
+                <var_name>__no_variable_name__</var_name>
+                <var_value>__no_variable_value__</var_value>
+            </var_000>
+        </variables>
+        <parameters>
+            <par_000>
+                <par_name>__no_parameter_name__</par_name>
+                <par_value>__no_parameter_value__</par_value>
+            </par_000>
+        </parameters>
         <action>
-            <script>initial_spikes_generator.py</script>
-            <parameters>
-                <argv_1_results_dir>CO_SIM_RESULTS_DIR</argv_1_results_dir>
-                <argv_2_output_file>init_spikes.npy</argv_2_output_file>
-            </parameters>
+            <performer>
+                <performer_binary>python3</performer_binary>
+                <performer_arguments>
+                    <argv_01_empty>{CO_SIM_EMPTY}</argv_01_empty>
+                </performer_arguments>
+            </performer>
```

Comment:
Right, I guess this is a good approach since on different systems/platforms, the python interpreter could required some specific parameters. Additionally, we could code different kind of "performers", meaning "routines" written on different programming languages, or a binary per se.